Content.Server/Storage/Components/ArtifactStorageComponent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Content.Server.Xenoarchaeology.XenoArtifacts;
+
+namespace Content.Server.Storage.Components;
+
+[RegisterComponent]
+public sealed class ArtifactStorageComponent : EntityStorageComponent
```

Comment:
I know, this war crime, but I can't bypass it without rewriting entity storage.

Reply:
sad